spec/workers/downstream_draft_worker_spec.rb

```diff
@@ -20,7 +21,11 @@
     stub_request(:put, %r{.*content-store.*/content/.*})
   end
 
-  describe "arguments" do
+  describe "initialisation" do
+    it "should have valid sidekiq options" do
+      expect(described_class).to have_valid_sidekiq_options
```

Comment:
Just copying a comment that Kevin made on my PR - this can be replaced by a 'specify' block and placed at the top of the tests, as per https://github.com/alphagov/link-checker-api/blob/5463ffdc9ac50b16ce1f340c8575031863dfd0ab/spec/workers/check_worker_spec.rb#L4 (same for `downstream_live_worker_spec.rb`)

Reply:
Thanks! I missed that line and thought you'd taken it out after testing 🤦. Will add now